internal-ingress-basic/README.md

```diff
@@ -0,0 +1,127 @@
+
+1. Download this repo and navigate to this folder
+
+```sh
+$ git clone https://github.com/mark-church/gke-networking-recipes
+Cloning into 'gke-networking-recipes'...
+
+$ cd gke-networking-recipes/internal-ingress-basic
+```
+
+2. Deploy the Ingress, Deployment, and Service resources in the [internal-ingress-basic.yaml](internal-ingress-basic.yaml) manifest.
+
+```sh
+$ kubectl apply -f internal-ingress-basic.yaml
+ingress.networking.k8s.io/foo-internal created
+service/foo created
+deployment.apps/foo created
+
+```
+
+
+3. It will take up to a minute for the Pods to deploy and up to a few minutes for the Ingress resource to be ready. Validate their progress and make sure that no errors are surfaced in the resource events.
+
+
+```
+$ kubectl get deploy foo
+NAME   READY   UP-TO-DATE   AVAILABLE   AGE
+foo    3/3     3            0           28s
+
+$ kubectl describe ingress foo-internal
+Name:             foo-internal
+Namespace:        default
+Address:          10.200.10.218
+Default backend:  default-http-backend:80 (10.200.20.7:8080)
+Rules:
+  Host             Path  Backends
+  ----             ----  --------
+  foo.example.com  
+                      foo:80 (<none>)
+Annotations:
+  ingress.kubernetes.io/target-proxy:                k8s2-tp-odw4qjdd-default-foo-internal-uw14fm1v
+  ingress.kubernetes.io/url-map:                     k8s2-um-odw4qjdd-default-foo-internal-uw14fm1v
+  kubectl.kubernetes.io/last-applied-configuration:  {"apiVersion":"networking.k8s.io/v1beta1","kind":"Ingress","metadata":{"annotations":{"kubernetes.io/ingress.class":"gce-internal"},"name":"foo-internal","namespace":"default"},"spec":{"rules":[{"host":"foo.example.com","http":{"paths":[{"backend":{"serviceName":"foo","servicePort":80}}]}}]}}
+
+  kubernetes.io/ingress.class:            gce-internal
+  ingress.kubernetes.io/backends:         {"k8s1-e4c0c439-default-foo-80-f3d4a56a":"Unknown","k8s1-e4c0c439-kube-system-default-http-backend-80-dee9de2e":"HEALTHY"}
+  ingress.kubernetes.io/forwarding-rule:  k8s2-fr-odw4qjdd-default-foo-internal-uw14fm1v
+Events:
+  Type     Reason     Age                  From                     Message
+  ----     ------     ----                 ----                     -------
+  Normal   ADD        23m                  loadbalancer-controller  default/foo-internal
+  Warning  Translate  23m                  loadbalancer-controller  error while evaluating the ingress spec: could not find service "default/foo"
+  Normal   CREATE     22m                  loadbalancer-controller  ip: 10.200.10.218
+  Normal   XPN        22m (x4 over 23m)    loadbalancer-controller  Firewall change required by network admin: `gcloud compute firewall-rules create k8s-fw-l7--e4c0c439b5effdb3 --network vpc-prod --description "GCE L7 firewall rule" --allow tcp:30000-32767,tcp:8080 --source-ranges 130.211.0.0/22,209.85.152.0/22,209.85.204.0/22,35.191.0.0/16 --target-tags gke-egr-airbus-test-cluster-1-f53dc7d4-node --project myproject`
```

Comment:
Did you test in a shared VPC host or service project? That might be why these messages show up since we haven't yet validated shared VPC support.